packages/api/methods.ts

```diff
@@ -21,7 +27,7 @@ export async function loadBudget(budgetId) {
   return send('api/load-budget', { id: budgetId });
 }
 
-export async function downloadBudget(syncId, { password } = {}) {
+export async function downloadBudget(syncId, password?) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏this would be a breaking change in the API. Do we really _need_ to do it?

Reply:
Good catch, there's no reason to change it atm. I will get it updated back to being proper.